src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,43 @@
+executor: qa-automation
+
+parameters:
+  test_path:
+    description: "The path to the test suite to run"
+    type: string
+  project_name:
+    description: "The name of the project running the tests"
```

Comment:
This parameter refers to the name of the project on Report Portal.

Reply:
☝️